test/test-elements.ts

```diff
@@ -0,0 +1,102 @@
+import {
+    customElement,
+    PropertyValues,
+    TemplateResult,
+    html,
+    property,
+} from 'lit-element';
+import { observable, computed } from 'mobx';
+import { MobxLitElement } from '../lit-mobx';
+import { nothing } from 'lit-html';
+
+@customElement('test-mobx-lit-element-updates-arrays')
+export class TestMobxLitElementUpdatesArrays extends MobxLitElement {
+    @property({ attribute: false })
+    @observable
+    public observableArray?: string[];
+
+    @property()
+    public normalProperty = 'My Array';
+
+    @computed
+    private get arrayLength(): string {
+        return `Array length = ${
+            this.observableArray ? this.observableArray.length : 'N/A'
+        }`;
+    }
+
+    public render(): TemplateResult {
+        return html`
+            <p>${this.normalProperty}</p>
+            <p>
+                ${this.observableArray
+                    ? this.observableArray.join(', ')
+                    : nothing}
+            </p>
+            <p>${this.arrayLength}</p>
+        `;
+    }
+}
+
+@customElement('test-mobx-lit-element-update-change')
+export class TestMobxLitElementUpdateChange extends MobxLitElement {
+    @observable
+    public observableValue: number = 1;
+
+    @observable
+    private derivedValue: number = 1;
+
+    public update(changedProperties: PropertyValues) {
+        super.update(changedProperties);
+        this.derivedValue = this.observableValue * 2;
+    }
+
+    public render(): TemplateResult {
+        return html`
+            <p>${this.observableValue}</p>
+            <p>${this.derivedValue}</p>
+        `;
+    }
+}
+
+@customElement('test-mobx-lit-element-updated-change')
+export class TestMobxLitElementUpdatedChange extends MobxLitElement {
+    @observable
+    public observableValue: number = 1;
+
+    @observable
+    private derivedValue: number = 1;
+
+    public updated(changedProperties: PropertyValues) {
+        super.updated(changedProperties);
```

Comment:
FYI, `updated` and `firstUpdated` don't take arguments, or have a super method you need to call in LitElement at least.

Reply:
Ah, I guess I don't need the super calls in the test components here that's true. I've gotten into the habit of making the super calls through mixin development, never know what you're being mixed into.